samples/assistant/csharp-ooproc/AssistantSample/AssistantSample.csproj

```diff
@@ -9,7 +9,7 @@
   <ItemGroup>
     <None Include="..\..\README.md" Link="README.md" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.21.0" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.2" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.1" />
```

Comment:
What was the reason to go backwards for this dependency?

Reply:
The docker build  for chat isolated sample throws an error with version 1.17.2, it does not pick the testversion of nuget package.
Hence, keeping the consolidated package at 1.17.1 for now.